### PR TITLE
Aleksei_Agishev

### PR DESCRIPTION
**Pull request by 50Leha**

### CODE REVIEW

File: gradlew.bat [hunk 1]
```diff
@@ -1,89 +1,100 @@
...
-rem Set variable GRADLE_EXIT_CONSOLE if you need the _script_ return code instead of
-rem the _cmd.exe /c_ return code!
-if  not "" == "%GRADLE_EXIT_CONSOLE%" exit 1
-exit /b 1
-
-:mainEnd
-if "%OS%"=="Windows_NT" endlocal
-
-:omega
+@rem
```
Comment (ArtemNikolaev1) [inline gradlew.bat]: Please merge with master and remove unnecessary fines from changes

File: gradlew.bat [hunk 2]
```diff
@@ -1,89 +1,100 @@
...
+
+set CMD_LINE_ARGS=%*
+
+:execute
+@rem Setup the command line
+
+set CLASSPATH=%APP_HOME%\gradle\wrapper\gradle-wrapper.jar
+
+@rem Execute Gradle
+"%JAVA_EXE%" %DEFAULT_JVM_OPTS% %JAVA_OPTS% %GRADLE_OPTS% "-Dorg.gradle.appname=%APP_BASE_NAME%" -classpath "%CLASSPATH%" org.gradle.wrapper.GradleWrapperMain %CMD_LINE_ARGS%
```
Comment (NikolaevArtem) [inline gradlew.bat]: If changes in .idea, gradlew and gradlew.bat are not needed, please revert them

File: src/main/java/homework_1/CommandLineArgs.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package homework_1;
+
+public class CommandLineArgs {
+    public static final String ANSI_RED = "\u001B[31m";
+    public static final String ANSI_RESET = "\u001B[0m";
+
+    public static void main(String[] args) {
+        for(String str: args){
+            if(!str.equals("ошибка")){
```
Comment (NikolaevArtem) [inline src/main/java/homework_1/CommandLineArgs.java]: Opt: better not use it this way, confusing

Comment (50Leha) [reply src/main/java/homework_1/CommandLineArgs.java]: what is exactly confusing: line 8 or line 9?
if you mean line 9, then please explain me, what`s wrong? As I figured out, it`s a normal practice to use it like that: https://stackoverflow.com/questions/8484668/java-does-not-equal-not-working

File: gradlew.bat (hunk 2 above)
Comment (50Leha) [reply gradlew.bat]: I merged it from master as you told me before. I haven`t worked with gradle yet, only maven. Therefore I`ve just merged it from master

File: src/main/java/homework_1/CommandLineArgs.java (hunk 3 above)
Comment (NikolaevArtem) [reply src/main/java/homework_1/CommandLineArgs.java]: You revert boolean value where it is not needed. You could use if(str.equals("ошибка")) {}

File: gradlew.bat (hunk 2 above)
Comment (NikolaevArtem) [reply gradlew.bat]: No, in master there are different files, that's why I can see changes here. Nevermind, I'll do it myself

File: src/main/java/homework_1/CommandLineArgs.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+                System.out.println(str + ": " + str.length());
```
Comment (NikolaevArtem) [inline src/main/java/homework_1/CommandLineArgs.java]: I think word "letters" is missing. Please read requirements

**Comment by NikolaevArtem** [comment]
Seems forgotten

**Pull request closed without merging**